.changeset/eight-tomatoes-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[fix] don't watch generated types directory
```

Comment:
The changeset police is here to remind you to update your changeset before you jubilantly/exhaustedly merge this.